main.js

```diff
@@ -0,0 +1,35 @@
+const menuIconMobile = document.querySelector('.navbar__hamb');
+const menu = document.querySelector('.navbar__menu');
+const viewport = document.querySelector('body');
+const menuLinks = document.querySelectorAll('.navbar__menu__link');
+const heroLanding = document.querySelector('.head');
+
+function isActiveClassExisted(element) {
+  return element.classList.contains('active');
+}
+
+function refreshIcon() {
+  if (isActiveClassExisted(menu)) {
+    menuIconMobile.setAttribute('src', 'images/hamburguer-close.png');
+  } else {
+    menuIconMobile.setAttribute('src', 'images/hamburguer.png');
+  }
+}
+function openMenu() {
+  menu.classList.toggle('active');
+  viewport.classList.add('blockover');
+  heroLanding.classList.add('blur');
+  refreshIcon();
+}
+
+function closeMenu() {
+  heroLanding.classList.remove('blur');
+  menu.classList.remove('active');
```

Comment:
-  [ ] Required: Kindly remove blur when the mobile menu is closed. When I close your mobile menu pressing ( X ). I am seeing blurr and cannot see the content until it is refreshed.
![2022-05-10 (2)](https://user-images.githubusercontent.com/70269507/167560656-e7503d10-cf14-42fd-8b88-62790c70f914.png)

Reply:
ok, I solve it , thanks for helpful advice

Reply:
already update linter.yml and aslo add my contributor information in readme file